frontend/src/components/MyReviewCard.js

```diff
@@ -56,12 +76,18 @@ const Wrapper = styled.section`
     white-space: nowrap;
     margin-bottom: 0;
   }
+
+  p: first-child {
```

Comment:
This looks weird.. either `p:first-child` or `p :first-child`, depending on what you want to target.